### ISSUE TITLE
Wireguard not working on reboot

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#1613]
It has been reported in our [forums](https://www.turnkeylinux.org/forum/support/tue-20210518-2012/turnkey-wireguard-works-properly-stops-working-after-restarting-vm) that the WireGuard appliance works fine, but following a reboot it no longer works.

Part of the issue (the `wg-quick@wg0.service`) is covered in #1610 but more is required.

It turns out that `net.ipv4.ip_forward` also needs to be enabled in `/etc/sysctl.conf`.
(see [this comment](https://www.turnkeylinux.org/forum/support/tue-20210518-2012/turnkey-wireguard-works-properly-stops-working-after-restarting-vm#comment-49803))

To summarize, this needs to be done to workaround (both of) the issue(s):
```
systemctl enable --now  wg-quick@wg0.service
sysctl -w net.ipv4.ip_forward=1
```

To fix the build code, this will probably do it:
```
systemctl enable wg-quick@wg0.service
sed -i "\|#* *net.ipv4.ip_forward=| s|^#* *\(net.ipv4.ip_forward *=\).*|\11|" /etc/sysctl.conf
```
(Note that the build code changes could also be applied to an existing VM, but it will require a reboot to start working).

**Comment by OnGle** [comment]
ping, what's the status on this issue @JedMeister?

**Comment by JedMeister** [comment]
Thanks for the bump @OnGle!
